src/3d/3dgraph.h

```diff
@@ -70,6 +70,8 @@ struct COLORS_IDS {
 		SKYFARMER_PIPETKA,
 		ROTTEN_ITEM,
 
+		BODY_CRIMSON,

```

Comment:
BODY_CRIMSON - what is it?

Reply:
A new material (red) for mechoses. Otherwise there was no way to add red color for them using the existing materials, as far as I can see. As for the name – had to name it as "body_crimson" because "body_red" is already in use for orange color. And, well, it is close enough in color.